infrastructure/api.tf

```
@@ -91,7 +91,28 @@ resource "aws_api_gateway_stage" "ndr_api" {
   deployment_id        = aws_api_gateway_deployment.ndr_api_deploy.id
   rest_api_id          = aws_api_gateway_rest_api.ndr_doc_store_api.id
   stage_name           = var.environment
-  xray_tracing_enabled = false
+  xray_tracing_enabled = var.enable_xray_tracing
+
+  depends_on = [aws_cloudwatch_log_group.api_gateway_stage]
+}
+
+resource "aws_cloudwatch_log_group" "api_gateway_stage" {
+  # Name must follow this format to allow execution logging 
+  # https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html
+  name              = "API-Gateway-Execution-Logs_${aws_api_gateway_rest_api.ndr_doc_store_api.id}/${var.environment}"
```

```suggestion
  name              = "${var.workspace}_API-Gateway-Execution-Logs"
```

Two things we don't want to create a new log group per deployment. Second better to use workspace than environment just because it will summarize all sandboxes as `dev`